src/styles/index.scss

```diff
@@ -1,3 +1,23 @@
+
+@import 'price';
+@import 'stars';
+@import 'product';
+@import 'button';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
+  display: inline-block;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+}
+
+.card {
```

Comment:
You should have only two blocks, use SASS nesting, move all elements of card block to one file, and only stars and index.scss should be in separate file.